server/tests/models/device_spec.js

```diff
@@ -0,0 +1,60 @@
+import chai, { expect } from 'chai';
```

Comment:
This file should be called device.test.js to stay consistent with the other tests (ticket_spec should also be changed)